distribution/docs/src/main/resources/content/_managing/_running/validator-plugins.adoc

```diff
@@ -0,0 +1,52 @@
+:title: Validator Plugins on Ingest
+:type: dataManagement
+:status: published
+:parent: Validating Data
+:order: 00
+:summary: Validator plugins run on Ingest
+
+== {title}
+
+On ingest, some validator plugins are run to ensure the data being ingested will be valid
```

Comment:
I think we can simplify this sentence. Maybe "When *Enforce Errors* is enabled within the ${admin-console}, validator plugins ensure the data being ingested is valid."

Reply:
fixed